examples/jsm/nodes/shadernode/ShaderNode.js

```diff
@@ -317,10 +317,22 @@ export function ShaderNode( jsFunc ) {
 export const nodeObject = ( val, altType = null ) => /* new */ ShaderNodeObject( val, altType );
 export const nodeObjects = ( val, altType = null ) => new ShaderNodeObjects( val, altType );
 export const nodeArray = ( val, altType = null ) => new ShaderNodeArray( val, altType );
-export const nodeProxy = ( ...val ) => new ShaderNodeProxy( ...val );
-export const nodeImmutable = ( ...val ) => new ShaderNodeImmutable( ...val );
+export const nodeProxy = ( ...params ) => new ShaderNodeProxy( ...params );
+export const nodeImmutable = ( ...params ) => new ShaderNodeImmutable( ...params );
+export const shader = ( jsFunc ) => new ShaderNode( jsFunc );
```

Comment:
Can we please add an empty line before this one, just for the ease of read? So `node*` go into one "block" and `shader` and `fn` in another.